### ISSUE TITLE
Text box in Notes

### ISSUE THREAD

**Issue by stothert** [archivesspace/archivesspace#1643]
Need functionality for expanding the content section. Difficult to compose or edit text without whole view. Also the typewriter font is disjointed from the interface and hard to read.

**Comment by LI-Yanlin** [comment]
I agree! Current box need at least a list of tags that is supported by the text box, I had to try them one by one and publish them, check them in the PUI. I also want to use TEI tags to mark things like typos in notes.
